apps/cowswap-frontend/src/modules/swap/pure/SwapButtons/index.tsx

```diff
@@ -83,6 +83,11 @@ const swapButtonStateMap: { [key in SwapButtonState]: (props: SwapButtonsContext
       <Trans>Error loading price. Try again later.</Trans>
     </GreyCard>
   ),
+  [SwapButtonState.QuoteExpired]: () => (
+    <GreyCard style={{ textAlign: 'center' }}>
+      <Trans>Quote expired. Refreshing...</Trans>
```

Comment:
When back to the page, shouldn't take long for the quote to be refreshed automatically